lib/semantics/check-call.cpp

```diff
@@ -587,6 +587,7 @@ static void CheckExplicitInterfaceArg(evaluate::ActualArgument &arg,
     const characteristics::DummyArgument &dummy,
     const characteristics::Procedure &proc, evaluate::FoldingContext &context,
     const Scope *scope) {
+  //  std::cout << "In CheckExplicitInterfaceArg\n";
```

Comment:
?

Reply:
Oops.  Thanks for catching this.  I'll remove it.